	here is a new patch file in order not to use sed;
% cat files/patch-configure
--- configure.orig	2003-12-28 08:53:30 UTC
+++ configure
@@ -3874,11 +3874,6 @@ for lt_ac_sed in $lt_ac_sed_list /usr/xp
   cat /dev/null > conftest.in
   lt_ac_count=0
   echo $ECHO_N "0123456789$ECHO_C" >conftest.in
-  # Check for GNU sed and select it if it is found.
-  if "$lt_ac_sed" --version 2>&1 < /dev/null | grep 'GNU' > /dev/null; then
-    lt_cv_path_SED=$lt_ac_sed
-    break
-  fi
   while true; do
     cat conftest.in conftest.in >conftest.tmp
     mv conftest.tmp conftest.in
